### ISSUE TITLE
rosboost_cfg can't find boost::system on Indigo

### ISSUE THREAD

**Issue by trainman419** [ros/ros#62]
As seen on ROS Answers: http://answers.ros.org/question/187492/xv_11_laser_driver-build-error/

```
CMake Error at /opt/ros/indigo/share/ros/core/rosbuild/public.cmake:847 (message):
  [rosboost-cfg --libs system] failed with error: Traceback (most recent call
  last):

    File "/opt/ros/indigo/bin/rosboost-cfg", line 35, in <module>
      rosboost_cfg.main()
    File "/opt/ros/indigo/lib/python2.7/dist-packages/rosboost_cfg/rosboost_cfg.py", line 359, in main
      output = lflags(ver, options.lflags.split(','))
    File "/opt/ros/indigo/lib/python2.7/dist-packages/rosboost_cfg/rosboost_cfg.py", line 278, in lflags
      s += lib_flags(ver, lib) + " "
    File "/opt/ros/indigo/lib/python2.7/dist-packages/rosboost_cfg/rosboost_cfg.py", line 268, in lib_flags
      lib = find_lib(ver, name)
    File "/opt/ros/indigo/lib/python2.7/dist-packages/rosboost_cfg/rosboost_cfg.py", line 238, in find_lib
      raise BoostError('Could not locate library [%s], version %s'%(name, ver))

  rosboost_cfg.rosboost_cfg.BoostError: "Could not locate library [system],
  version (1, 54, 0, '/usr', '/usr/include', True, True)"

Call Stack (most recent call first):
  CMakeLists.txt:29 (rosbuild_link_boost)
```

**Comment by dirk-thomas** [comment]
Under Saucy / Trusty Boost libraries are installed to `/usr/lib/x86_64-linux-gnu` and are therefore not found by the dry rosboost_cfg script.

**Comment by dirk-thomas** [comment]
Fix released in ros-indigo-ros 1.11.4.
